datanode/api/trading_data_v2.go

```diff
@@ -2644,6 +2644,31 @@ func (t *tradingDataServiceV2) proposalToGovernanceData(ctx context.Context, pro
 	return &gd, nil
 }
 
+func (t *tradingDataServiceV2) GetVotesByProposal(ctx context.Context, req *v2.GetVotesByProposalRequest) (*v2.GetVotesByProposalResponse, error) {
```

Comment:
I'm not sure if this endpoint has any value, you can get the same data returned by calling `GetGovernanceData()` and passing in a proposal ID?